templates/zerver/realm_creation_disabled.html

```diff
@@ -0,0 +1,13 @@
+{% extends "zerver/portico.html" %}
+
+{% block for_you %}{{ _('got a bit lost there.') }}{% endblock %}
+
+{% block portico_content %}
+
+<br/>
+<p class="lead">{{ _('Realm creation disabled') }}</p>
```

Comment:
@timabbott I think it's confusing that some times we refer this process as creating new organization and sometimes as team creation. Should I stick with one of it?